apps/fyler/src/fyler_category_manager.erl

```diff
@@ -0,0 +1,694 @@
+-module(fyler_category_manager).
+-behaviour(gen_fsm).
+
+-include("../include/log.hrl").
+-include("fyler.hrl").
+
+-define(REGEX, "\"PrivateIpAddress\": \"(?<ip>[^\"]*)\"").
+-define(NODE(Category, Ip), list_to_atom(lists:flatten(io_lib:format("fyler_~p_pool@~s", [Category, Ip])))).
```

Comment:
Line 8 is too long: -define(NODE(Category, Ip), list_to_atom(lists:flatten(io_lib:format("fyler_~p_pool@~s", [Category, Ip]))))..